recipes/infer-subc/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "infer-subc" %}
+{% set version = "0.0.post1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/infer-subc-{{ version }}.tar.gz
+  sha256: 7ff6222f124d7167c3334e25d0178adc8d0205b3f5f189fb7ebc4e34379959a8
+
+build:
+  entry_points:
+    - infer_subc = infer_subc.__main__:main
```

Comment:
This file is a dummy script, it seems?

https://github.com/ndcn/infer-subc/blob/5e084d437add5d823ea45bebbb63742bb8b000e6/infer_subc/cli.py#L28

Reply:
Indeed, should we remove it then?